src/main/java/com/google/ehub/servlets/ItemPageServlet.java

```diff
@@ -88,4 +88,18 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       }
     }
   }
+
+  @Override
+  public void doDelete(HttpServletRequest request, HttpServletResponse response)
+      throws IOException {
+    long commentId;
+    CommentDataManager comment = new CommentDataManager();
```

Comment:
nit: you can move this line closer to when CommentDataManager is actually used, on line 103.